src/ttsnake.c

```diff
@@ -266,7 +270,7 @@ void showscene (char scene[][NROWS][NCOLS], int number, int menu)
       printf ("Score: %.2f\r\n", score);
       printf ("Blocks: %d\r\n", block_count);  
 	  
-      printf ("Controls: q: quit\t\r\n");
+      printf ("Controls: q: quit\t\r\n\t  r: restart\t\r\n");
```

Comment:
In my terminal, having `r` in a new line causes the game scene to scroll up, and the top border is missed.
Perhaps limiting the controls to a single line might fix this.

Reply:
I couldn't replicate but it is in a single line now.
```printf ("Controls: q: quit | r: restart\r\n");```

Reply:
@monacofj It must be specific to your screen size. In any case, it looks good now in a single line.